docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -56,8 +57,7 @@ OpenMP Policies
 
 OpenMP Target Policies
 ^^^^^^^^^^^^^^^^^^^^^^^^
-* ``omp_target_parallel_for_exec<NUMTEAMS>`` - Execute a loop in parallel using an ``omp target parallel for`` pragma with given number of thread teams; e.g.,
-if a GPU device is available, this is similar to launching a CUDA kernel with 
+* ``omp_target_parallel_for_exec<NUMTEAMS>`` - Execute a loop in parallel using an ``omp teams distribute parallel for num_teams(NUMTEAMS)`` pragma with given number of thread teams inside a ``omp target`` region; e.g., if a GPU device is available, this is similar to launching a CUDA kernel with 
```

Comment:
"NUMTEAMS" should be "ThreadsPerTeam", but I can update this and the following sentence when I finalize the omp target issue.

Reply:
I left it that way because your PR has not been merged yet.  Do you think it will be ready to go in by Friday?  We need to get this release out by the end of the week so it is available for folks who attend the ECP tutorial next week.